src/api/app/jobs/report_to_scm_job.rb

```diff
@@ -8,7 +8,7 @@ def perform(event_id)
 
     EventSubscription.joins('INNER JOIN events ON event_subscriptions.eventtype = events.eventtype AND event_subscriptions.package_id = events.package_id')
                      .where(events: { eventtype: ['Event::BuildFail', 'Event::BuildSuccess'], id: event_id }, channel: :scm)
-                     .where(Event.arel_table[:undone_jobs].gt(0))
+                     .where(Event::Base.arel_table[:undone_jobs].gt(0))
```

Comment:
We always forget the `::Base`, but somehow it worked before because we tested that exact line in the Rails console. Anyway, I'm fine with that change.

Reply:
```
[1] pry(main)> Event.respond_to?(:arel_table)
=> false
[2] pry(main)> Event::Base.respond_to?(:arel_table)
=> true
[3] pry(main)> 
```
but yes, I remember that we tested it together.. who knows